code/lib/channels/README.md

```diff
@@ -31,6 +31,8 @@ class Transport {
 
 Currently, channels are baked into storybook implementations and therefore this module is not designed to be used directly by addon developers. When developing addons, use the `getChannel` method exported by `@storybook/addons` module. For this to work, Storybook implementations should use the `setChannel` method before loading addons.
 
+TODO: this needs to be updated to reflect the manager/preview split
+
 ```js
 import { addons } from '@storybook/addons';
 
```

Comment:
I'm going to delete this code, we should document how to get the channel in the preview and manager somewhere @jonniebigodes, do we have a good place for that?